.github/workflows/publish-helm-chart.yaml

```diff
@@ -16,22 +16,22 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 #v2.4.0
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579  # v2.4.0
```

Comment:
Why adding golangci-linter has anything to do with Helm chart workflow?

Reply:
It is a fix for ibiqlik/action-yamllint@v3 warnings

Reply:
Ah I see. Please check also my comment about golangci-lint.yml that you added and reusing build.yaml instead